docs/configuration.md

```diff
@@ -1474,6 +1474,13 @@ Apart from these, the following properties are also available, and may be useful
     the new Kafka direct stream API.
   </td>
 </tr>
+<tr>
+  <td><code>spark.streaming.ui.retainedBatches</code></td>
+  <td>1000</td>
```

Comment:
I think the only question is whether this was intended to be exposed to users or not. What's your argument for that? This should be 100, and changed to 1000 in your other PR if this goes in.